src/bitcoin-cli.cpp

```diff
@@ -867,6 +870,86 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * ParseGetInfoResult takes in -getinfo result in UniValue object and parses it
+ * into a user friendly UniValue string to be printed on the console.
+ * @param[out] result  Reference to UniValue result containing the -getinfo output.
+ */
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) return;
+
+#ifndef WIN32
+    const std::string RESET = "\x1B[0m";
+    const std::string GREEN = "\x1B[32m";
+    const std::string BLUE = "\x1B[34m";
+    const std::string YELLOW = "\x1B[33m";
+    const std::string MAGENTA = "\x1B[35m";
+    const std::string CYAN = "\x1B[36m";
+#else
+    const std::string RESET = "";
+    const std::string GREEN = "";
+    const std::string BLUE = "";
+    const std::string YELLOW = "";
+    const std::string MAGENTA = "";
+    const std::string CYAN = "";
+#endif
+    const std::string BITCOIN_EMOJI = "\u20BF";
+
+    std::string result_string = strprintf("%sChain: %s%s\n", BLUE, result["chain"].getValStr(), RESET);
+    result_string += strprintf("Blocks: %s\n", result["blocks"].getValStr());
+    result_string += strprintf("Headers: %s\n", result["headers"].getValStr());
+    result_string += strprintf("Verification progress: %s\n", result["verificationprogress"].getValStr());
+    result_string += strprintf("Difficulty: %s\n\n", result["difficulty"].getValStr());
+
+    result_string += strprintf(
+        "%sNetwork: in %s, out %s, total %s%s\n",
+        GREEN,
+        result["connections"]["in"].getValStr(),
+        result["connections"]["out"].getValStr(),
+        result["connections"]["total"].getValStr(),
+        RESET);
+    result_string += strprintf("Version: %s\n", result["version"].getValStr());
+    result_string += strprintf("Time offset (s): %s\n", result["timeoffset"].getValStr());
+    result_string += strprintf("Proxy: %s \n", result["proxy"].getValStr());
+    result_string += strprintf("Relay fee (%\\/kB): %s\n\n", CURRENCY_UNIT, result["relayfee"].getValStr());
+
+    if (!result["has_wallet"].isNull()) {
+        const std::string walletname = result["walletname"].getValStr();
+        result_string += strprintf("%sWallet: %s%s\n", MAGENTA, walletname.empty() ? "\"\"" : walletname, RESET);
```

Comment:
I think that instead of `""`, it is more common to use `[default wallet]`. 

At least these make me think so:

* https://github.com/bitcoin/bitcoin/blob/ea8b2e8e127b5a27424c1c08a1352ec1c9382c96/src/qt/bitcoingui.cpp#L401
* https://github.com/bitcoin/bitcoin/pull/13275/

Reply:
In the CLI the default wallet has been `""`.

Reply:
Is that user-friendly? I mean `""` looks like an output for programmers and not for users. However, if it is a convention in CLI, then it's better to align with it.

____
One can possibly write also `default wallet ("")`

Reply:
If you need to specify the default wallet in the CLI or RPC, you pass `""` or just nothing, e.g.:
```
$ bitcoin-cli -signet -rpcwallet="" -getinfo
$ bitcoin-cli -signet -rpcwallet=  -getinfo
```
If you pass something else, it won't be recognized as the default wallet (unless there is some way I'm not aware of)

Reply:
I see, that makes sense. Even though I was not suggesting to pass "default wallet" to a CLI command, just to make it clear that `""` means a default wallet in the text output. 

Anyway, all is good.